health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/setupCampaignHelpers.js

```
@@ -157,14 +157,14 @@ export const cycleDataRemap=(data)=> {
   
   }
   
-  export const  processDelivery=(delivery, resourcesMap, ageInfo , type)=> {
+  export const  processDelivery=(delivery, resourcesMap, ageInfo , type ,projectType)=> {
   
     return {
       id: parseInt(delivery.deliveryIndex, 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)